### ISSUE TITLE
Parser error while linting an empty HTML file

### ISSUE THREAD

**Issue by gebsh** [angular-eslint/angular-eslint#66]
**Repro**
```JS
module.exports = {
  overrides: [
    {
      files: ['*.component.html'],
      parser: '@angular-eslint/template-parser',
      plugins: ['@angular-eslint/template'],
      rules: {
        '@angular-eslint/template/banana-in-a-box': 'error',
      },
    },
  ],
};
```

And an empty HTML file `src/app/app.component.html`.

**Expected Result**
Do not throw an error after running `eslint src/app/app.component.html`.

**Actual Result**
```
/home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html
  0:0  error  Parsing error: Cannot read property 'start' of undefined
```

**Additional Info**
There's no error when I add some content to the file. Linting also works fine after adding some invalid code:

```HTML
<div ([foo])="test"></div>
```
```
/home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html
  1:6  error  Invalid binding syntax. Use [(expr)] instead  @angular-eslint/template/banana-in-a-box

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Addtional output of `eslint src/app/app.component.html --debug`:
<details>
  <summary>Details</summary>

```
  eslint:cli CLI args: [ 'src/app/app.component.html', '--debug' ] +0ms
  eslint:cli Running on files +4ms
  eslint:config-array-factory Loading JSON config file: /home/gebsh/Projects/tests/eslint/test-eslint/package.json +0ms
  eslint:ignore-pattern Create with: [ IgnorePattern { patterns: [ '/node_modules/*', '/bower_components/*' ], basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', loose: false } ] +0ms
  eslint:ignore-pattern   processed: { basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', patterns: [ '/node_modules/*', '/bower_components/*' ] } +2ms
  eslint:ignore-pattern Create with: [ IgnorePattern { patterns: [ '/node_modules/*', '/bower_components/*' ], basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', loose: false } ] +0ms
  eslint:ignore-pattern   processed: { basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', patterns: [ '/node_modules/*', '/bower_components/*' ] } +1ms
  eslint:file-enumerator Start to iterate files: [ 'src/app/app.component.html' ] +0ms
  eslint:file-enumerator File: /home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html +1ms
  eslint:cascading-config-array-factory Load config files for /home/gebsh/Projects/tests/eslint/test-eslint/src/app. +0ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh/Projects/tests/eslint/test-eslint/src/app. +0ms
  eslint:config-array-factory Config file not found on /home/gebsh/Projects/tests/eslint/test-eslint/src/app +4ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh/Projects/tests/eslint/test-eslint/src. +0ms
  eslint:config-array-factory Config file not found on /home/gebsh/Projects/tests/eslint/test-eslint/src +0ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh/Projects/tests/eslint/test-eslint. +0ms
  eslint:config-array-factory Loading JS config file: /home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js +0ms
  eslint:config-array-factory Config file found: /home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js +1ms
  eslint:config-array-factory Loading parser "@angular-eslint/template-parser" from /home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js +2ms
  eslint:config-array-factory Loaded: @angular-eslint/template-parser@0.0.1-alpha.27 (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/@angular-eslint/template-parser/dist/index.js) +0ms
  eslint:config-array-factory Loading plugin "@angular-eslint/template" from /home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js +63ms
  eslint:config-array-factory Loaded: @angular-eslint/eslint-plugin-template@0.0.1-alpha.27 (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/@angular-eslint/eslint-plugin-template/dist/index.js) +1ms
  eslint:config-array-factory Plugin /home/gebsh/Projects/tests/eslint/test-eslint/node_modules/@angular-eslint/eslint-plugin-template/dist/index.js loaded in: 221ms +221ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh/Projects/tests/eslint. +288ms
  eslint:config-array-factory Loading package.json config file: /home/gebsh/Projects/tests/eslint/package.json +0ms
  eslint:config-array-factory Loading JSON config file: /home/gebsh/Projects/tests/eslint/package.json +0ms
  eslint:config-array-factory Error reading package.json file: /home/gebsh/Projects/tests/eslint/package.json +0ms
  eslint:config-array-factory Config file not found on /home/gebsh/Projects/tests/eslint +0ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh/Projects/tests. +1ms
  eslint:config-array-factory Config file not found on /home/gebsh/Projects/tests +1ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh/Projects. +0ms
  eslint:config-array-factory Config file not found on /home/gebsh/Projects +0ms
  eslint:cascading-config-array-factory No cache found: /home/gebsh. +0ms
  eslint:cascading-config-array-factory Stop traversing because of considered root. +0ms
  eslint:cascading-config-array-factory Configuration was determined: ConfigArray(3) [ { name: 'DefaultIgnorePattern', filePath: '', criteria: null, env: undefined, globals: undefined, ignorePattern: IgnorePattern { patterns: [Array], basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', loose: false }, noInlineConfig: undefined, parser: undefined, parserOptions: undefined, plugins: undefined, processor: undefined, reportUnusedDisableDirectives: undefined, root: undefined, rules: undefined, settings: undefined }, { name: '.eslintrc.js', filePath: '/home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js', criteria: null, env: undefined, globals: undefined, ignorePattern: undefined, noInlineConfig: undefined, parser: undefined, parserOptions: undefined, plugins: undefined, processor: undefined, reportUnusedDisableDirectives: undefined, root: undefined, rules: undefined, settings: undefined }, { name: '.eslintrc.js#overrides[0]', filePath: '/home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js', criteria: { includes: [Array], excludes: null, basePath: '/home/gebsh/Projects/tests/eslint/test-eslint' }, env: undefined, globals: undefined, ignorePattern: undefined, noInlineConfig: undefined, parser: { error: null, filePath: '/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/@angular-eslint/template-parser/dist/index.js', id: '@angular-eslint/template-parser', importerName: '.eslintrc.js#overrides[0]', importerPath: '/home/gebsh/Projects/tests/eslint/test-eslint/.eslintrc.js' }, parserOptions: undefined, plugins: { '@angular-eslint/template': [Object] }, processor: undefined, reportUnusedDisableDirectives: undefined, root: undefined, rules: { '@angular-eslint/template/banana-in-a-box': 'error' }, settings: undefined } ] on /home/gebsh/Projects/tests/eslint/test-eslint/src/app +1ms
  eslint:ignore-pattern Create with: [ IgnorePattern { patterns: [ '/node_modules/*', '/bower_components/*' ], basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', loose: false } ] +293ms
  eslint:ignore-pattern   processed: { basePath: '/home/gebsh/Projects/tests/eslint/test-eslint', patterns: [ '/node_modules/*', '/bower_components/*' ] } +0ms
  eslint:ignore-pattern Check {
  filePath: '/home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html',
  dot: false,
  relativePath: 'src/app/app.component.html',
  result: false
} +0ms
  eslint:cli-engine Lint /home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html +0ms
  eslint:linter Linting code for /home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html (pass 1) +0ms
  eslint:linter Verify +0ms
  eslint:linter With ConfigArray: /home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html +0ms
  eslint:linter Parsing error: Cannot read property 'start' of undefined
  eslint:linter TypeError: Cannot read property 'start' of undefined
    at Object.f [as parseForESLint] (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/@angular-eslint/template-parser/dist/index.js:1:1944)
    at parse (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/linter/linter.js:640:22)
    at Linter._verifyWithoutProcessors (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/linter/linter.js:1111:33)
    at Linter._verifyWithConfigArray (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/linter/linter.js:1255:21)
    at Linter.verify (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/linter/linter.js:1210:25)
    at Linter.verifyAndFix (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/linter/linter.js:1400:29)
    at verifyText (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/cli-engine/cli-engine.js:230:48)
    at CLIEngine.executeOnFiles (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/cli-engine/cli-engine.js:798:28)
    at Object.execute (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/lib/cli.js:212:111)
    at Object.<anonymous> (/home/gebsh/Projects/tests/eslint/test-eslint/node_modules/eslint/bin/eslint.js:107:28) +3ms
  eslint:linter Generating fixed text for /home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html (pass 1) +0ms
  eslint:source-code-fixer Applying fixes +0ms
  eslint:source-code-fixer shouldFix parameter was false, not attempting fixes +0ms
  eslint:file-enumerator Complete iterating files: ["src/app/app.component.html"] +296ms
  eslint:cli-engine Linting complete in: 297ms +3ms

/home/gebsh/Projects/tests/eslint/test-eslint/src/app/app.component.html
  0:0  error  Parsing error: Cannot read property 'start' of undefined

✖ 1 problem (1 error, 0 warnings)
```
</details>

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/builder` | `0.0.1-alpha.27` |
| `@angular-eslint/eslint-plugin`        | `0.0.1-alpha.27` |
| `@angular-eslint/eslint-plugin-template`        | `0.0.1-alpha.27` |
| `@angular-eslint/template-parser`        | `0.0.1-alpha.27` |
| `TypeScript`                       | `3.8.3` |
| `ESLint`                           | `6.8.0` |
| `node`                             | `13.12.0` |
| `npm`                              | `6.14.4` |

**Comment by JamesHenry** [comment]
Thanks for the report @gebsh I can reproduce within the integration-tests workspace.

Do you fancy opening a PR to address this?

**Comment by gebsh** [comment]
Hey James, I think I can have a look at this.

**Comment by JamesHenry** [comment]
Fixed in 0.0.1-alpha.29
